app/views/ConfirmationView.scala.html

```diff
@@ -91,5 +94,9 @@
         @p(Html(Messages(breakdownHelper.eligibility(gfp))))
     }
 
+    @p(Html(Messages("confirmation.p5")))
+
+    @p(Html(link(routes.FeedbackSurveyController.startSurvey(), "confirmation.print.label", true)))
```

Comment:
i guess this link is wrong for the print

Reply:
yeah dummy links for now